### PR TITLE
add label when check the approvals

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: remote/mock/remote.go [hunk 1]
```diff
@@ -1,17 +1,29 @@
-package mock
```
Comment (tboerger) [inline remote/mock/remote.go:1]: Is it possible that your mock generator lib is an older version?

File: web/hook.go [hunk 2]
```diff
@@ -78,13 +82,34 @@ func Hook(c *gin.Context) {
...
+	var idx = len(approvers)
+	if idx > 2 {
+		idx = 2
+	}
+
+	var ns = make([]string, len(labels))
+	copy(ns, labels)
+
+	// remove old labels
+	err = remote.RemoveIssueLabels(c, user, repo, hook.Issue.Number, append(ns[:idx], ns[idx+1:]...))
```
Comment (tboerger) [inline web/hook.go:102]: This will remove the labels on every webhook?

**Review by tboerger — COMMENTED** [review]
I think the code needs to be not intelligent, otherwise it will always delete and add the labels.

File: remote/mock/remote.go (hunk 1 above)
Comment (lunny) [reply remote/mock/remote.go:1]: I followed the readme installation, don't know

File: web/hook.go (hunk 2 above)
Comment (lunny) [reply web/hook.go:102]: Remove old label if lgtm status changed.

**Review by tboerger — APPROVED** [review]

**Comment by tboerger** [comment]
LGTM

**Comment by xinity** [comment]
LGTM

**Pull request merged**